assets/css/bootstrap.css

```diff
@@ -1484,7 +1484,7 @@ blockquote ul:last-child,
 blockquote ol:last-child {
   margin-bottom: 0;
 }
-blockquote p { 
+blockquote * + p { 
   padding-bottom: 1em; 
```

Comment:
I think it has to be `padding-top` as we're modding the second element in a pair